getting-started-gradle/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'java'
+    id 'io.quarkus'
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
```

Comment:
I think we could use our BOM, it would be better. Not sure how Gradle works with BOM, though?